emails/tests/fixtures/s3_stored_replies_expected.email

```diff
@@ -0,0 +1,29 @@
+Subject: Re: Test Mozilla User New Domain Address
+From: a1b2c3d4@test.com
+To: sender@external.example.com
+Reply-To: a1b2c3d4@test.com
```

Comment:
question (non-blocking): When/why do we have `test.com` in the `Reply-To` header sometimes, and other times we have `default.com` in the `Reply-To` header?

Reply:
Good question, looking into it...

Reply:
Here's the `Reply-To` header for each:

- `domain_recipient_expected.email` - `Reply-To: replies@default.com`
- `s3_stored_replies_expected.email` - `Reply-To: a1b2c3d4@test.com`
- `single_recipient_expected.email` - `Reply-To: replies@default.com`
- `single_recipient_list_expected.email` - `Reply-To": "replies@default.com`

The `s3_stored_replies_expected.email` is a reply from the Relay user back to the external email. So, if the external sender wants to reply, they should reply to the email mask. For tests, that domain is `test.com`.

https://github.com/mozilla/fx-private-relay/blob/29512b38ad47e1103a9cbdc683b3e28888824b6d/emails/models.py#L40-L47